src/modules/fw_path_navigation/FixedwingPositionControl.cpp

```diff
@@ -2550,6 +2553,10 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const float control_interva
 	const float throttle_trim_comp = compensateTrimThrottleForDensityAndWeight(_param_fw_thr_trim.get(), throttle_min,
 					 throttle_max);
 
+	const Vector3f local_pos_a = Vector3f{_local_pos.ax, _local_pos.ay, _local_pos.az};
+	const Vector3f local_pos_v = Vector3f{_local_pos.vx, _local_pos.vy, _local_pos.vz};
+	const float acceleration_in_direction_of_flight = local_pos_v.unit_or_zero().dot(local_pos_a);
```

Comment:
Why into direction of flight? don't we want in direction of airspeed vector? - so subtracting the wind estimate off the ground vel vector.

Reply:
@tstastny @sfuhrer That would at least in theory give you the correct "airspeed derivative". When you fly a circle with wind then the vehicle is constantly accelerating / decelerating  depending on where it is on the circle. Measuring the inertial acceleration into the wind direction should more or less cancel out the acceleration as the relative wind vector is more or less constant in magnitude, right?

Reply:
@sfuhrer Have you tried this solution in SITL with wind as well?

Reply:
Yes I probably neglected wind a bit too much.
Anyway, I'm currently looking into removing the acceleration input into TECS completely, as before the fix https://github.com/PX4/PX4-Autopilot/pull/20443/commits/b1cbe04694bcb85161126df641dc7ebb66e8697a got in that was roughly how TECS functioned (the internal TAS_rate state was always close to 0). And it flew just fine, actually better then it does now.

Reply:
> Anyway, I'm currently looking into removing the acceleration input into TECS completely, as before the fix https://github.com/PX4/PX4-Autopilot/commit/b1cbe04694bcb85161126df641dc7ebb66e8697a got in that was roughly how TECS functioned (the internal TAS_rate state was always close to 0). And it flew just fine, actually better then it does now.

Should we just zero the accel input for now then, e.g. for 1.14 release, such that there is no change in behavior / regression - then we re-include accel once we've had time to make sure it is fedback correctly and tested?